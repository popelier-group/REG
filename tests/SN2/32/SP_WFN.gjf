# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.51889900   -0.00006800   -0.00009000
 H                 -1.18737800   -0.87463200   -0.54541700
 H                 -1.18731300    0.90944000   -0.48485100
 H                 -1.18732200   -0.03503900    1.02994700
 Br                 1.77118200    0.00001300    0.00003900
 Cl                -3.37334600    0.00000100   -0.00004800

frame.wfn
