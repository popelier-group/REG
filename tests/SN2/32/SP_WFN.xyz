
6

C       -1.375738    0.022480    0.176884
H       -0.944450   -0.529990    1.002062
H       -1.170761   -0.465713   -0.767467
H       -1.025561    1.046912    0.169302
Br        1.887061   -0.030907   -0.242553
Cl       -3.214820    0.052686    0.413185
