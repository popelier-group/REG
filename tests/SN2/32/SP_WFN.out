 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.32/Gau-173407.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.32/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=    173408.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.5189   -0.00007  -0.00009 
 H                    -1.18738  -0.87463  -0.54542 
 H                    -1.18731   0.90944  -0.48485 
 H                    -1.18732  -0.03504   1.02995 
 Br                    1.77118   0.00001   0.00004 
 Cl                   -3.37335   0.       -0.00005 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.518899   -0.000068   -0.000090
      2          1           0       -1.187378   -0.874632   -0.545417
      3          1           0       -1.187313    0.909440   -0.484851
      4          1           0       -1.187322   -0.035039    1.029947
      5         35           0        1.771182    0.000013    0.000039
      6         17           0       -3.373346    0.000001   -0.000048
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.082659   0.000000
     3  H    1.082658   1.785100   0.000000
     4  H    1.082655   1.785130   1.785120   0.000000
     5  Br   3.290081   3.132986   3.132869   3.132840   0.000000
     6  Cl   1.854447   2.416788   2.416786   2.416779   5.144528
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.375738    0.022480    0.176884
      2          1           0       -0.944450   -0.529990    1.002062
      3          1           0       -1.170761   -0.465713   -0.767467
      4          1           0       -1.025561    1.046912    0.169302
      5         35           0        1.887061   -0.030907   -0.242553
      6         17           0       -3.214820    0.052686    0.413185
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.3616513           0.7442407           0.7442406
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       162.6754463194 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.81D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40321816     A.U. after   11 cycles
            NFock= 11  Conv=0.99D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55384-101.41666 -62.20679 -56.02461 -56.02453
 Alpha  occ. eigenvalues --  -56.02453 -10.08931  -9.33084  -8.41544  -7.09468
 Alpha  occ. eigenvalues --   -7.08601  -7.08601  -6.23850  -6.23816  -6.23816
 Alpha  occ. eigenvalues --   -2.35086  -2.35076  -2.35076  -2.35049  -2.35049
 Alpha  occ. eigenvalues --   -0.71610  -0.54780  -0.48687  -0.31248  -0.31247
 Alpha  occ. eigenvalues --   -0.27875  -0.18119  -0.18119  -0.05250  -0.05087
 Alpha  occ. eigenvalues --   -0.05087
 Alpha virt. eigenvalues --    0.10700   0.11903   0.14620   0.14620   0.17418
 Alpha virt. eigenvalues --    0.18006   0.19441   0.19441   0.20561   0.20561
 Alpha virt. eigenvalues --    0.21540   0.22196   0.22196   0.24695   0.27845
 Alpha virt. eigenvalues --    0.30313   0.30314   0.36088   0.40148   0.53083
 Alpha virt. eigenvalues --    0.53083   0.54936   0.58250   0.58250   0.61179
 Alpha virt. eigenvalues --    0.62056   0.62057   0.64161   0.76050   0.76051
 Alpha virt. eigenvalues --    0.79421   0.84216   0.88048   0.88049   0.89201
 Alpha virt. eigenvalues --    0.89202   0.94420   0.96045   0.96045   1.00461
 Alpha virt. eigenvalues --    1.00461   1.01831   1.04859   1.25595   1.25595
 Alpha virt. eigenvalues --    1.26244   1.56488   1.67061   1.67061   1.84239
 Alpha virt. eigenvalues --    1.84240   1.85815   2.10482   2.21746   2.21747
 Alpha virt. eigenvalues --    2.26576   2.26577   2.27018   2.45105   2.45105
 Alpha virt. eigenvalues --    2.54582   2.58518   2.63537   2.63538   2.66127
 Alpha virt. eigenvalues --    2.66128   3.07310   3.28659   3.86499   3.86503
 Alpha virt. eigenvalues --    4.39764   4.39767   4.40338   4.40339   4.48334
 Alpha virt. eigenvalues --    6.75365   7.80402   7.80403   7.84739   9.83500
 Alpha virt. eigenvalues --   24.02327  25.98071  25.98072  26.48061  48.06859
 Alpha virt. eigenvalues --  215.82056 290.01216 290.01216 290.053181020.82340
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.588220   0.399835   0.399832   0.399825  -0.042945  -0.097048
     2  H    0.399835   0.517484  -0.037641  -0.037633   0.003666  -0.038382
     3  H    0.399832  -0.037641   0.517468  -0.037633   0.003664  -0.038373
     4  H    0.399825  -0.037633  -0.037633   0.517467   0.003661  -0.038369
     5  Br  -0.042945   0.003666   0.003664   0.003661  35.922215   0.043957
     6  Cl  -0.097048  -0.038382  -0.038373  -0.038369   0.043957  17.164314
 Mulliken charges:
               1
     1  C   -0.647720
     2  H    0.192672
     3  H    0.192683
     4  H    0.192682
     5  Br  -0.934217
     6  Cl   0.003900
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.069683
     5  Br  -0.934217
     6  Cl   0.003900
 Electronic spatial extent (au):  <R**2>=           1278.4014
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -4.9406    Y=              0.0808    Z=              0.6351  Tot=              4.9819
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -76.3679   YY=            -44.5060   ZZ=            -45.0240
   XY=              0.5222   XZ=              4.0962   YZ=             -0.0671
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -21.0686   YY=             10.7933   ZZ=             10.2753
   XY=              0.5222   XZ=              4.0962   YZ=             -0.0671
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             43.4095  YYY=              0.4403  ZZZ=             -3.4129  XYY=              8.5611
  XXY=             -0.4451  XXZ=             -3.3915  XZZ=              8.8001  YZZ=             -0.9909
  YYZ=             -1.0063  XYZ=             -0.0697
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1690.8583 YYYY=            -77.1003 ZZZZ=           -100.9099 XXXY=             14.5498
 XXXZ=            113.8767 YYYX=             11.1063 YYYZ=             -1.4230 ZZZX=             94.0083
 ZZZY=             -1.8537 XXYY=           -268.4888 XXZZ=           -274.8996 YYZZ=            -29.6679
 XXYZ=             -0.6386 YYXZ=             31.1182 ZZXY=              4.8053
 N-N= 1.626754463194D+02 E-N=-7.678584738810D+03  KE= 3.070006919951D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE776\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.518899,-0.000068,-0.00009\H,0,-1.187378,-0.874632,-0.545417\H,0
 ,-1.187313,0.90944,-0.484851\H,0,-1.187322,-0.035039,1.029947\Br,0,1.7
 71182,0.000013,0.000039\Cl,0,-3.373346,0.000001,-0.000048\\Version=ES6
 4L-G16RevC.01\State=1-A\HF=-3074.4032182\RMSD=9.928e-09\Dipole=-1.9600
 468,-0.000055,-0.0000926\Quadrupole=-16.061798,8.0308475,8.0309505,-0.
 0000243,-0.0003779,-0.0000035\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 UNDER MOST GREENE GRASSE LY MOST GREATE SNAKES.
 -- PROVERBS AND REASONS OF THE YEAR 1585
           AS REPRINTED IN PAISLEY MAGAZINE 1828.
 Job cpu time:       0 days  0 hours  0 minutes 17.7 seconds.
 Elapsed time:       0 days  0 hours  0 minutes  9.2 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 19:03:12 2020.
